packages/cli/lib/projects.js

```diff
@@ -168,14 +168,51 @@ const validateProjectConfig = (projectConfig, projectDir) => {
   }
 };
 
+const pollFetchProject = async (accountId, projectName) => {
+  // Temporary solution for gating slowness. Retry on 403 statusCode
+  try {
+    const project = await fetchProject(accountId, projectName);
+    return project;
+  } catch (error) {
+    if (error.statusCode === 404) {
+      throw error;
+    }
+  }
+  return new Promise((resolve, reject) => {
+    let pollCount = 0;
+    const spinnies = SpinniesManager.init();
+    spinnies.add('pollFetchProject', {
+      text: 'Fetching project status',
+    });
+    const pollInterval = setInterval(async () => {
+      try {
+        const project = await fetchProject(accountId, projectName);
+        if (project) {
+          spinnies.remove('pollFetchProject');
```

Comment:
Rather than succeed, I thought removing would be cleaner since we already have a warning message: 
<img width="1108" alt="Screenshot 2023-05-09 at 15 20 39" src="https://github.com/HubSpot/hubspot-cli/assets/16788677/d0f8559b-3919-4052-81bc-af1aaef185a2">

Let me know what you think